dataland-frontend/tests/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../tsconfig.cypress-e2e.json"
+}
```

Comment:
Heißt das die config soll die selben Werte übernehmen (bzw. eigentlich ergänzen), die in der anderen config stehen?